ssh/tailssh/incubator.go

```diff
@@ -312,6 +290,143 @@ func beIncubator(args []string) error {
 	return err
 }
 
+// tryLoginShell attempts to handle the ssh session by creating a full login
+// shell. If it was able to do so, it returns true plus any error from running
+// that shell. If it was unable to do so, it returns false, nil.
```

Comment:
i don't understand, what does `true, error` mean? log in failed?

Reply:
true means we tried to run `login` or `su`, err means that it failed.